ci/examples/agent.d/test-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  server_attributes: {
+    attribute1: "value1",
+    attribute2: "value2",
```

Comment:
Just pointing to the unused `attribute2`, in case if there was a plan to use it somehow :)

Reply:
yeah didn't plan to use it, since asserting on 1 attribute is sufficient